dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -185,6 +189,7 @@ function jumpout ()
    jumps_permitted = jumps_permitted - 1
    last_sys = system.cur()
    if not job_done and last_sys == missys then
+      misn.appendLog("Mission failed, pirate escaped")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.